src/components/select/select.jsx

```diff
@@ -135,7 +135,7 @@ const Select = forwardRef(function Select(props, ref) {
   }, [value]);
 
   function handleBlur(event) {
-    if (!(listBoxRef.current && listBoxRef.current.contains(event.relatedTarget))) {
+    if (!(refs.listbox.current && refs.listbox.current.contains(event.relatedTarget))) {
```

Comment:
having a `!` outside of a wrapped `&` makes this hard to tell what is actually intended. I know this was already here mostly, but personally I'd take the time to invert the logic slightly:

```suggestion
    if (!refs.listbox.current || !refs.listbox.current.contains(event.relatedTarget))) {
```

Reply:
Gotta get those parentheses matching bruda

Reply:
psh, never take a suggestion without proofreading!